test/integration/targets/ansible-galaxy/runme.sh

```diff
@@ -195,6 +199,13 @@ f_ansible_galaxy_status \
 
     [[ $(grep -c 'ansible_test\.airport' out.txt) -eq 1 ]]
 
+f_ansible_galaxy_status \
+    "collection list specific collection which contains galaxy.yml"
+
+    ansible-galaxy collection list -p ./install ansible_test.development | tee out.txt
+
+    [[ $(grep -c 'ansible_test\.development' out.txt) -eq 1 ]]
+
```

Comment:
This test won't catch the warning that shouldn't be there.
```suggestion
    ansible-galaxy collection list -p ./install ansible_test.development 2>&1 | tee out.txt

    [[ $(grep -c 'ansible_test\.development' out.txt) -eq 1 ]]
    [[ $(grep -c 'WARNING' out.txt) -eq 0 ]]

```